lookout/style/format/tests/test_robustness.py

```diff
@@ -50,18 +49,20 @@ def test_style_robustness_report(self):
         self.assertGreater(metrics["recall"], 0)
         self.assertGreater(metrics["precision"], 0)
 
-    @unittest.skipIf(sys.version_info.minor == 5, "Python 3.5 is not yet supported"
-                                                  " by difflib")
+    @unittest.skipIf(sys.version_info.minor == 5, "Python 3.5 is not yet supported by difflib")
     def test_plot_pr_curve(self):
         with tempfile.NamedTemporaryFile(prefix="output-figure", suffix=".png") as tmpf:
             with Capturing() as output:
-                plot_pr_curve(true_repo=self.input_pattern,
-                              noisy_repo=self.input_pattern_noisy,
-                              bblfsh=self.bblfsh,
-                              language=self.language,
-                              model_path=self.model_path,
-                              support_threshold=0,
-                              output=tmpf.name)
+                try:
+                    plot_pr_curve(true_repo=self.input_pattern,
+                                  noisy_repo=self.input_pattern_noisy,
+                                  bblfsh=self.bblfsh,
+                                  language=self.language,
+                                  model_path=self.model_path,
+                                  support_threshold=0,
+                                  output=tmpf.name)
+                except SystemExit:
+                    self.skipTest("Matplotlib is required to run this test")
```

Comment:
I think we should not include `matplotlib` into docker image, so I just skip this test in case we do not have it.
Otherwise, the test is failed.